pkg/sortition/internal/local/chain.go

```diff
@@ -27,7 +27,15 @@ type Chain struct {
 	eligibleStake      map[chain.Address]*big.Int
 	eligibleStakeMutex sync.RWMutex
 
-	isPoolLocked bool
+	operatorRewards      map[chain.Address]*Rewards
+	operatorRewardsMutex sync.RWMutex
+
+	isPoolLocked     bool
+	currentTimestamp *big.Int
+}
+
+type Rewards struct {
+	ineligibleUntil *big.Int
 }
```

Comment:
I've created this struct with only one field because `Rewards.sol` has 3 additional fields: `accumulated, available, and weight`. I thought that down the road we will add more functionalities around these fields and it will be easier to add them in this struct and actually mimic the smart contract. Are you sure we won't add anything around `accumulated, available, and weight`?

Reply:
Yes, the client does not care about rewards.

Reply:
Done in [d3da44a](https://github.com/keep-network/keep-core/pull/3085/commits/d3da44a6fa9390cd8c49863a37546b2e4a8505f4)